Magento_CatalogWidget/templates/product/widget/content/grid.phtml

```diff
@@ -319,4 +319,13 @@ use Magento\Framework\App\Action\Action;
 
         <?= $block->getPagerHtml() ?>
     </div>
+    <?php if($block->getBlockHtml('formkey')): ?>
+        <script type="text/x-magento-init">
+            {
+                ".block.widget [data-role=tocart-form]": {
```

Comment:
We don't use .block and .widget classes in our theme. This should be `section__content--products`